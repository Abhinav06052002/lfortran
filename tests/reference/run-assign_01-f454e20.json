{
    "basename": "run-assign_01-f454e20",
    "cmd": "lfortran --continue-compilation --no-color {infile}",
    "infile": "tests/errors/assign_01.f90",
    "infile_hash": "b3907bfc11ab4a5400099590f3d20715a7fed312e26241facae934fb",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "run-assign_01-f454e20.stdout",
    "stdout_hash": "d27559a4e4d37b05997e0672e5314635e81b58d6eca9e3a76dd378ad",
    "stderr": "run-assign_01-f454e20.stderr",
    "stderr_hash": "22926a0ca854e3a26213cee88788e52c189e7e6d275089ff0cdc971c",
    "returncode": 1
}