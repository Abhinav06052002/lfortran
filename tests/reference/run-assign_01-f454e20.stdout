(TranslationUnit
    (SymbolTable
        1
        {
            assign_01:
                (Program
                    (SymbolTable
                        2
                        {
                            x:
                                (Variable
                                    2
                                    x
                                    []
                                    Local
                                    (IntegerConstant 2 (Integer 4) Decimal)
                                    (IntegerConstant 2 (Integer 4) Decimal)
                                    Parameter
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    assign_01
                    []
                    [(Assignment
                        (Var 2 x)
                        (IntegerConstant 1 (Integer 4) Decimal)
                        ()
                    )
                    (Print
                        (StringFormat
                            ()
                            [(Var 2 x)]
                            FormatFortran
                            (String -1 0 () PointerString)
                            ()
                        )
                    )]
                )
        })
    []
)
